src/rabbitmanager.js

```diff
@@ -113,6 +113,28 @@ class RabbitManager {
     vhost = encodeURIComponent(vhost);
     await this.request(`permissions/${vhost}/${user}`, {method: 'DELETE'});
   }
+
+  async queues() {
+    return await this.request('queues');
+  }
+
+  async queue(name, vhost='/') {
+    vhost = encodeURIComponent(vhost);
+    return await this.request(`queues/${vhost}/${name}`);
```

Comment:
Won't this default to a string like `queues///<name>`? I assume that's not desired.